### PR TITLE
RNs 4.6.22

### PR DESCRIPTION
**Pull request by stevsmit**
Adds RN 4.6.22 to be merged on 3-23. 

Preview at https://deploy-preview-30693--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-22

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
...
+
+link:https://access.redhat.com/solutions/5894581[{product-title} 4.6.22 container image list]
+
+[id="ocp-4-6-22-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Zeroconf library did not correctly rate-limit multicast DNS (mDNS) responses. As a result, excessive mDNS traffic flooded network records. This update adds rate-limiting to the Zeroconf library, which significantly reduces mDNS traffic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1936539[*BZ#1936539*])
++
+* Previously, an API server could fail to create a resource, which would return a 409 status code when there was a conflict updating a resource quota. Consequently, the resource would fail to create, and you might have had to retry the API request. With this update, the OpenShift Console web application attempts to retry the request 3 times when receiving a 409 status code, which is often sufficient for completing the request. In the event that a 409 status code continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928230[*BZ#1938230*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2683]: s/OpenShift Console web application/OpenShift web console
s/3 times/three times

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
...
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/5894581[{product-title} 4.6.22 container image list]
+
+[id="ocp-4-6-22-bug-fixes"]
+==== Bug fixes
+
+* Previously, the Zeroconf library did not correctly rate-limit multicast DNS (mDNS) responses. As a result, excessive mDNS traffic flooded network records. This update adds rate-limiting to the Zeroconf library, which significantly reduces mDNS traffic. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1936539[*BZ#1936539*])
++
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2682]: No need for the `+` characters between independent bullets.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
...
+* Previously, an API server could fail to create a resource, which would return a 409 status code when there was a conflict updating a resource quota. Consequently, the resource would fail to create, and you might have had to retry the API request. With this update, the OpenShift Console web application attempts to retry the request 3 times when receiving a 409 status code, which is often sufficient for completing the request. In the event that a 409 status code continues to occur, an error will be displayed in the console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1928230[*BZ#1938230*])
++
+* Previously, the livenessProbe would fail when the Prometheus container was under a heavy load (for example, during WAL replay). This heavy load would cause several issues and endless restart loops. This update removes livenessProbe, and as a result, heavy loads no longer cause endless restart loops. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1935586[*BZ#1935586*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2685]: - s/livenessProbe/liveness probe
- We typically try to stay away from parentheses. Maybe `...heavy load; for example, during WAL replay.`
- s/WAL/write-ahead logging (WAL)

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
...
++
+* Due to iptables rewriting rules, clients that used a fixed source port to connect to a service via both the service IP and a pod IP might have encountered problems with port conflicts. With this update, an additional OVS rule is inserted to notice when port conflicts occur and to do an extra SNAT to avoid said conflicts. As a result, there are no longer port conflicts when connecting to a service. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937547[*BZ#1937547*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2687]: s/OVS/Open vSwitch (OVS)
s/SNAT/source network address translation (SNAT)

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -2663,3 +2663,34 @@ link:https://access.redhat.com/solutions/5873402[{product-title} 4.6.21 containe
...
++
+* Previously, nodes were marked as `Ready` and would admit pods before they had a chance to sync. Consequently, `Pod status` might have gone out of sync, and oftentimes many were stuck in `nodeAffinity` at startup if the node was not cordoned. In this update, nodes are not marked as `Ready` until the node has synced with PI servers at least once. As a result, pods will no longer get stuck in `nodeAffinity` after a cold cluster restart.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930960[*BZ#1930960*])
++
+* Previously, when upgrading a cluster from a prior version with an idled workload, the idled workload would not wake on HTTP request once upgraded to {product-title} 4.6/4.7 due to `oc idle` feature fix-ups and reworks. With this update, idling changes are mirrored from endpoints to services on Ingress Operator startup. As a result, unidling workloads after upgrades works as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1927364[*BZ#1927364*])
```
Comment (codyhoag) [inline release_notes/ocp-4-6-release-notes.adoc:2691]: s/would not wake on HTTP request/would not start on HTTP request
s/feature fix-ups and reworks/feature updates

**Review by codyhoag — COMMENTED** [review]
Just a few nits; LGTM otherwise!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (stevsmit) [reply release_notes/ocp-4-6-release-notes.adoc]: All errors have been fixed!

**Pull request merged**